docs/reference/content/bson/pojos.md

```diff
@@ -0,0 +1,262 @@
++++
+date = "2017-04-10T14:27:51-04:00"
+title = "POJOs"
+[menu.main]
+  parent = "BSON"
+  weight = 45
+  pre = "<i class='fa fa-star'></i>"
++++
+
+## POJOs - Plain Old Java Objects
+
+The 3.5 release of the driver adds POJO support via the [`PojoCodec`]({{<apiref "org/bson/codecs/pojo/PojoCodec.html">}}), which allows for 
+direct serialization of POJOs to and from BSON. Internally, each `PojoCodec` utilizes a
+[`ClassModel`]({{<apiref "org/bson/codecs/pojo/ClassModel.html">}}) instance to store metadata about how the POJO should be serialized.
+
+A `ClassModel` for a POJO includes:
+
+  * The class of the POJO.
+  * A new instance factory. Handling the creation of new instances of the POJO. By default it requires the POJO to have an empty constructor.
+  * Field information, a list of [`FieldModel`]({{<apiref "org/bson/codecs/pojo/FieldModel.html">}}) instances that contain all the field metadata.
+  * An optional IdField. By default the `_id` or `id` field in the POJO.
+  * Type data for the POJO and it's fields to work around type erasure.
+  * An optional discriminator value. The discriminator is the value used to represent the POJO class being stored and by default is the 
```

Comment:
This is confusing.  It says the discriminator is optional, then that by default is the simple class name.  IIRC by default there is no discriminator.